### DISCUSSION TITLE
volume bar and brightness bar disappear

### DISCUSSION THREAD

**Discussion by qoocharly** [valinet/ExplorerPatcher#1810] [Q&A]
hello everyone. Recently I 've installed KB5029263. When I adjust the brightness and the volume, volume bar and brightness bar disappear on my laptop, although the the brightness and the volume did change. I tried to uninstall the "ExplorerPatcher" then it become normal. Does anyone have any idea?
Thank you in advance.

**Comment by TrainzC99** [comment]
Exactly the same for me.

Although I have discovered that if you switch back to the default Windows 11 Taskbar the volume and brightness bars come back.
You can do this from Properties >> Taskbar >> Taskbar style and select Windows 11.

○ Explorer Patcher Info:
Version		22621.1992.56.1

○ OS Info:
Edition		Windows 11 Pro
Version		22H2
Installed on	‎15/‎07/‎2023
OS build		22621.2134
Experience	Windows Feature Experience Pack 1000.22659.1000.0

**Reply by qoocharly** [reply]
Thanks
The problem can indeed be solved, but I'm not very accustomed to the style of Windows 11.
It seems it's the only solution.
